cmd/nodecmd/create.go

```diff
@@ -54,6 +54,7 @@ will apply to all nodes in the cluster`,
 		Args:         cobra.ExactArgs(1),
 		RunE:         createNode,
 	}
+	cmd.Flags().BoolVar(&useEIP, "use-elastic-ip", true, "attach Elastic IP on AWS coud servers")
```

Comment:
if we want to continue with previous CLI behaviour, the user should be asked for the option to either use elastic or not, unless a flag for that is given